core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -23,6 +23,7 @@ class PolicyManager {
 
     var freePolicies = 0
     var storedCulture = 0
+    var totalCulture = 0 // for contests
```

Comment:
I'd prefer if the name of this variable also showed it has something to do with contests, and maybe it should be saved in diplomacymanagers anyway, as it is only ever used for city state contests?

Reply:
ok, moved it to CivilizationInfo.